html/speaker-test.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Speaker Test</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    </head>
+
+    <body>
+        {{template "navbar"}}
+
+
+        <div class="container">
+            <div class="container pt-5 pl-0">
+                <h2>Speaker Test<br></h2>
+            </div>
+            <hr>
+
+            <div class="row">
+                <!-- <div class="col-md-5" >
+                    <div class="card mb-3">
+                        <div class="card-body">
+                            <h5 class="card-title"><strong>Speaker</strong></h5>
+                        </div>
+                    </div>
+                </div> -->
+                <div class="col-md-4">
+                    <div class="card mb-3">
+                        <div class="card-body">
+                            <!-- <h5 class="card-title"><strong>Status</strong></h5> -->
+                            <button id="speaker-test-button" type="button" onclick="speakerTest()" class="btn btn-primary mb-4">Play</button>
+                            <p style="white-space: pre-wrap;" id="speaker-test-statustext"></p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-8">
+                    <div class="card mb-3">
+                        <div class="card-body">
+                            <h5 class="card-title"><strong>Output</strong></h5>
+                            <h4 id="speaker-test-indicator">&#x003F;</h4>
+                            <p style="white-space: pre-wrap;" id="speaker-test-outputext"></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <script src="/static/js/jquery-3.3.1.slim.min.js"></script>
+        <script src="/static/js/bootstrap.min.js"></script>
+        <script>
+          function speakerTest() {
+            // Set status indicator
+            var statusIndicator = document.getElementById("speaker-test-indicator");
+            statusIndicator.textContent = "...";
+            // Clear output text
+            var statusText = document.getElementById("speaker-test-statustext");
+            statusText.textContent = "Sound file is now playing.";
+            var outputText = document.getElementById("speaker-test-outputext");
+            outputText.textContent = "";
+            // Disable test button while waiting for response
+            var button = document.getElementById("speaker-test-button");
+            button.disabled = true;
+            console.log("/speaker-status/")
```

Comment:
delete

Reply:
Hmmm, ok, I assumed I had to disable it so that it wouldn't be pressed again whilst the sound is still playing.

Will delete those lines and test.

Reply:
Disabling the button is good! Just the `console.log` line.